### DISCUSSION TITLE
If something is bothering you.

### DISCUSSION THREAD

**Discussion by JENOT-ANT** [JENOT-ANT/gravity-simulation#6] [Q&A] [answered]
This tab, as the name suggests, is for questions about the project. So if you have any questions, or you want to clarify something, feel free to start a new discussion hire.

**Comment by sumanthumesh** [comment, marked as answer]
I tried running temporary-solar-system-simulation.py. I have attached the screenshot of what I see. The camera movement and zoom in/out functions seem to work fine. But the objects are in the same place and do not move. I am new to pygame, so are the objects (rectangles) supposed to move in the example or not? 

![image](https://user-images.githubusercontent.com/39519813/173627889-be3cc21a-cdf1-4b03-aa51-5ffefb5ec80d.png)

**Reply by JENOT-ANT** [reply]
Actually in this simulation everything is in the huge scale. If you zoom enough (to see round planets), for example onto second rectangle (Mercury), you should see the movement. If you want to make this simulation faster, I can show you where you can change it inside code, it's the matter of one or two numbers.
If you have some problems of course feel free to ask.

**Reply by JENOT-ANT** [reply]
- So now I've updated this file. I've added some constants (FRAME_RATE and ANIMATION_SPEED) on the top of it. If you would see faster some effects, you can change those variables (especially ANIMATION_SPEED) to higher values.
- You can also click 'R' to stop clearing display, so there should be objects' trace visible.
- We're going to upload some other examples to.
